core/src/main/java/com/redhat/lightblue/client/request/metadata/MetadataCreateSchemaRequest.java

```diff
@@ -22,5 +26,23 @@ public String getOperationPathParam() {
     public HttpMethod getHttpMethod() {
         return HttpMethod.PUT;
     }
+    
+    
+    @Override
+    public String getRestURI(String baseServiceURI) {
+    	StringBuilder requestURI = new StringBuilder();
+
+        requestURI.append(baseServiceURI);
+
+        if (StringUtils.isNotBlank(this.getEntityName())) {
+            appendToURI(requestURI, this.getEntityName());
+        }
+
+    	if (StringUtils.isNotBlank(this.getEntityVersion())) {
+            appendToURI(requestURI, this.schema +"="+ this.getEntityVersion());
+        }
```

Comment:
This won't do. Schema is in request body, not part of uri. Parse string to json, and set the request body.

Also, overload the schema constructor to get a JsonNode schema, because body is json in AbstractListhblueMEtadataRequest